k9mail-library/src/test/java/com/fsck/k9/mail/store/imap/ImapFolderTest.java

```diff
@@ -1228,7 +1256,7 @@ private void assertCommandWithIdsIssued(String expectedCommand) throws Messaging
             String command = commandPrefixes.get(i) +
                     " " + ImapUtility.join(",", commandUids.get(i)) + " " +
                     commandSuffixes.get(i);
-            if (command.equals(expectedCommand)) {
+            if (command.trim().equals(expectedCommand)) {
```

Comment:
I'd rather fix the construction of `command` above than trimming the erroneously added space at the end.